src/PIL/ImageDraw.py

```diff
@@ -373,6 +375,16 @@ def multiline_text(
         stroke_width=0,
         stroke_fill=None,
     ):
+        if direction == "ttb":
+            raise ValueError("ttb direction is unsupported for multiline text")
```

Comment:
I didn't implement anchor handling for vertical text due to #4945, so instead this raises an error to avoid confusion due to unexpected anchor values.